package/libretro-prboom/libretro-prboom.mk

```diff
@@ -21,9 +21,12 @@ define LIBRETRO_PRBOOM_BUILD_CMDS
 endef
 
 define LIBRETRO_PRBOOM_INSTALL_TARGET_CMDS
-    mkdir -p "${BINARIES_DIR}/retroarch/cores"
+	mkdir -p "${BINARIES_DIR}/retroarch/cores"
+	mkdir -p "${BINARIES_DIR}/retroarch/system/prboom"
 	$(INSTALL) -D $(@D)/prboom_libretro.so \
 		${BINARIES_DIR}/retroarch/cores/prboom_libretro.so
+	$(INSTALL) -D $(@D)/prboom.wad \
+		${BINARIES_DIR}/retroarch/system/prboom
```

Comment:
There is no possibility to select wad file from hidden directory

Reply:
No, if you load this wad it starts shareware doom

Reply:
Interesting, it does not start for me, also that's what written in manual.

Reply:
Indeed, maybe I messed something up.

Reply:
I think no harm leaving it in ./system (it silences coreinfo warning), but if you want to cp it somewhere else it's ok.